src/appmixer/mailchimp/bundle.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "appmixer.mailchimp",
-  "version": "2.0.5",
+  "version": "2.0.6",
```

Comment:
@sayam-nasir  we are introducing a new feature - new port, it ought to be "minor" release (2.1.0)